test-partner/Dockerfile

```diff
@@ -38,6 +43,13 @@ RUN \
 # Add go and oc binary directory to $PATH
 ENV PATH="/bin/":"/usr/local/go/bin":${GOPATH}"/bin/":"${PATH}"
 
+# Configure SSH daemon
+WORKDIR /home/${USERNAME}/sshd
+RUN \
+	printf 'Port 2222\nHostKey /home/%s/sshd/ssh_host_rsa_key\nPidFile /home/%s/sshd/sshd.pid\n' "$USERNAME" "$USERNAME" >> sshd_config \
```

Comment:
What does this line mean?

Reply:
It creates an `sshd_config` file with some options that seem to be required when running the SSH daemon as an unprivileged user. The port 2222 is just a personal choice, it can be overwritten with the -p option when launching the SSH daemon.